app/models/rent.rb

```diff
@@ -8,4 +8,5 @@ class Rent < ApplicationRecord
   def to_cannot_be_lesser_than_from
     errors.add(:to, "can't be lesser than from") if to < from
   end
+  include Filterable
```

Comment:
I would put this at the top of the model, so it's more visible. You can always have your own order of includes validates and relationships so it's easier to read when there are lots of stuff in the model. Separating this kinds of things with blank lines is also a great thing. Just think how would you like someone new to the code to read this and understand what the class does better.